.github/workflows/diff.yaml

```diff
@@ -345,19 +345,15 @@ jobs:
 
       - name: Build release binaries
         run: |
-          # Activate toolchain.
-          source /opt/toolchain-v4/activate
-          # Initialize dependencies.
-          ./init
-          # Build only memgraph release binarie.
-          cd build
-          cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
-          make -j$THREADS memgraph
+          ./release/binary/run.sh build-builder
+          ./release/binary/run.sh run-builder 
+          ./release/binary/run.sh build-memgraph 
+          ./release/binary/run.sh copy-binary memgraph-binary 
 
       - name: Run Jepsen tests
         run: |
           cd tests/jepsen
-          ./run.sh test-all-individually --binary ../../build/memgraph --ignore-run-stdout-logs --ignore-run-stderr-logs
+          ./run.sh test-all-individually --binary ../../memgraph-binary --ignore-run-stdout-logs --ignore-run-stderr-logs
```

Comment:
Is it possible to still copy data from the build folder, e.g. just making a new symlink from `build/memgraph` to `../../memgraph-binary`?

Anyway, two things:
* try to make it work if people have local binary because that's suitable for development / testing
* have everything under build (any git ignored folder, will make sure the binary doesn't have to ignored or uploaded accidentally)

Reply:
- It is possible to use any local binary to test
- I will move everything under build :+1: